.gitignore

```diff
@@ -13,3 +13,4 @@ _site
 .jekyll-cache
 .asciidoctor
 verification-results
+bin/
```

Comment:
This does not seem to be related to the commit? I would make it a separate commit then and explain for which case you ignore it (I guess you're using Eclipse? Or what tool creates a `bin` folder?)

Reply:
Indeed using Eclipse based IDE; was accidental that this was included, sorry about that! Will revert.

Reply:
No problem :slightly_smiling_face: You can also still add a commit where you add it with an explanation for the next Eclipse user, if you want. I mean, it's not harmful :man_shrugging:

Reply:
Pushed now as a separate commit.